### PR TITLE
[CircleGraph] Support tflite

### PR DESCRIPTION
**Pull request by glistening**
Now, CircleGraph can show tflite.
It also updates circle-schema. It shows more operators like `GELU`.

ONE-vscode-DCO-1.0-Signed-off-by: Sanggyu Lee <sg5.lee@samsung.com>

### CODE REVIEW

File: src/OneExplorer/ConfigObject.ts [hunk 1]
```diff
@@ -301,6 +301,17 @@ export class ConfigObj {
...
+      artifactAttr: {
+        ext: ".tflite",
+        icon: new vscode.ThemeIcon("symbol-variable"),
+        openViewType: "one.viewer.circle",
+      },
+      locator: new Locator((value: string) =>
+        LocatorRunner.searchWithExt(".tflite", value)
+      ),
+    });
+
```
Comment (dayo09) [inline src/OneExplorer/ConfigObject.ts:314]: This one should go to LINE 219-227

**Comment by glistening** [comment]
Resolved conflict with #1656.

**Comment by glistening** [comment]
I see several `OneExplorer` errors from `Check PR test`. I excludes the change in `OneExplorer`, which can be done separately. I can see both `circle` and `tflite` w/o `OneExplorer` change.

File: media/CircleGraph/external/circle-schema.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-var $root = flatbuffers.get('circle');
+var cir_root = flatbuffers.get('circle');
```
Comment (glistening) [inline media/CircleGraph/external/circle-schema.js:1]: All `$root` is replaced with `cir_root` to avoid conflict with `$root` from `tflite-schema.js`.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by dayo09 — APPROVED** [review]
LGTM Thanks!

**Pull request merged**